components/formats-bsd/src/loci/formats/AxisGuesser.java

```diff
@@ -71,18 +73,24 @@
   public static final int S_AXIS = 4;
 
   /** Prefixes indicating space dimension. */
-  public static final String[] Z_PREFIXES = {
+  public static final ImmutableSet<String> Z_PREFIXES = ImmutableSet.of(
```

Comment:
Not to harp, but we should make sure that having a google object in the public API is something we want since we won't be able to change the implementation without a breaking change.